quickwit-doc-mapper/src/tokenizers.rs

```diff
@@ -48,3 +156,189 @@ fn raw_tokenizer_test() {
+            "1331901000.000000",
+            "CHEt7z3AzG4gyCNgci",
+            "192.168.202.79",
+            "50465",
+            "192.168.229.251",
+            "80",
+            "1",
+            "HEAD",
+            "192.168.229.251",
+            "/DEASLog02.nsf",
+            "Mozilla/5.0",
+        ];
+
+        let mut token_stream = get_quickwit_tokenizer_manager()
+            .get("log")
+            .unwrap()
+            .token_stream(test_string);
+
+        array_ref.iter().for_each(|ref_token| {
+            if token_stream.advance() {
+                assert_eq!(&token_stream.token().text, ref_token)
+            } else {
+                panic!()
+            }
+        });
+    }
+
+    #[test]
+    fn log_tokenizer_log_test_http() {
+        let test_string = "{\"message\" : \"211.11.9.0 - - [1998-06-21T15:00:01-05:00] \"GET \
+                           /english/index.html HTTP/1.0\" 304 0\"}";
+        let array_ref: [&str; 8] = [
+            "message",
+            "211.11.9.0",
+            "1998-06-21T15:00:01-05:00",
+            "GET",
+            "/english/index.html",
+            "HTTP/1.0",
+            "304",
+            "0",
+        ];
+        let mut token_stream = get_quickwit_tokenizer_manager()
+            .get("log")
+            .unwrap()
+            .token_stream(test_string);
+
+        array_ref.iter().for_each(|ref_token| {
+            if token_stream.advance() {
+                assert_eq!(&token_stream.token().text, ref_token)
+            } else {
+                panic!()
+            }
+        });
+    }
+
+    #[test]
+    fn log_tokenizer_log_wsa() {
+        let test_string = "54.36.149.41 - - [22/Jan/2019:03:56:14 +0330] \"GET /filter/27|13%20%D9%85%DA%AF%D8%A7%D9%BE%DB%8C%DA%A9%D8%B3%D9%84,27|%DA%A9%D9%85%D8%AA%D8%B1%20%D8%A7%D8%B2%205%20%D9%85%DA%AF%D8%A7%D9%BE%DB%8C%DA%A9%D8%B3%D9%84,p53 HTTP/1.1\" 200 30577 \"-\" \"Mozilla/5.0 (compatible; AhrefsBot/6.1; +http://ahrefs.com/robot/)\" \"-\"";
+
+        let array_ref: [&str; 16] = [
+            "54.36.149.41",
+            "22/Jan/2019:03:56:14",
+            "0330",
+            "GET",
+            "/filter/27",
+            "13%20%D9%85%DA%AF%D8%A7%D9%BE%DB%8C%DA%A9%D8%B3%D9%84",
+            "27",
+            "DA%A9%D9%85%D8%AA%D8%B1%20%D8%A7%D8%B2%205%20%D9%85%DA%AF%D8%A7%D9%BE%DB%8C%DA%A9%D8%\
+             B3%D9%84",
+            "p53",
+            "HTTP/1.1",
+            "200",
+            "30577",
+            "Mozilla/5.0",
+            "compatible",
+            "AhrefsBot/6.1",
+            "http://ahrefs.com/robot/",
+        ];
+
+        let mut token_stream = get_quickwit_tokenizer_manager()
+            .get("log")
+            .unwrap()
+            .token_stream(test_string);
+
+        array_ref.iter().for_each(|ref_token| {
+            if token_stream.advance() {
+                assert_eq!(&token_stream.token().text, ref_token)
+            } else {
+                panic!()
+            }
+        });
```

Comment:
Since this snippet is being reused, you can write a helper function. Also, you can merge the tests into a single one. Check out `quickwit-config/src/templating.rs`